packages/@react-spectrum/badge/chromatic/Badge.chromatic.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Badge} from '../';
+import CheckmarkCircle from '@spectrum-icons/workflow/src/CheckmarkCircle';
+import {Flex} from '@react-spectrum/layout';
+import {Meta, Story} from '@storybook/react';
+import React from 'react';
+import {SpectrumBadgeProps} from '@react-types/badge';
+import {Text} from '@react-spectrum/text';
+
+let variants = ['positive', 'info', 'negative', 'neutral', 'yellow', 'fuchsia', 'indigo', 'seafoam', 'magenta', 'purple'];
+
+const meta: Meta<SpectrumBadgeProps> = {
+  title: 'Badge',
+  component: Badge
+};
+
+export default meta;
+
+const Template = (): Story<SpectrumBadgeProps> => (args) => (
```

Comment:
haha, almost, this is CSF2

3 is what you used for the stories file down below

Reply:
lol i keep referencing the wrong examples 😭